src/azure-cli/azure/cli/command_modules/rdbms/custom.py

```diff
@@ -87,7 +140,39 @@ def _server_create(cmd, client, resource_group_name, server_name, sku_name, no_w
             location=location,
             tags=tags)
 
-    return client.create(resource_group_name, server_name, parameters)
+    server_result = resolve_poller(
```

Comment:
What does resolve_poller do here? Is create a long running operation here?

Reply:
Yes.

Reply:
How long does it take for the operation? Does customer need to wait for a long time?
Could we not wait for LRO complete?

Reply:
It takes around 10 minutes for the operation to complete. Yes, the customer would need to wait otherwise if we return them the serverdetails w/o waiting for the operation to be complete, then they won't be be able to connect to the server if the server creation fails.